lib/promiscuous_black_hole/db.rb

```diff
@@ -16,6 +16,11 @@ def self.search_path
       fetch('show search_path').first[:search_path]
     end
 
+    def self.max_identifier_length
+      @max_identifier_length ||=
+        fetch('show max_identifier_length').first[:max_identifier_length].to_i - 1
```

Comment:
no idea why i need to do 1 less than the max here. i can manually create a table at the maximum length just fine, but this failed otherwise. maybe something internal to the sequel gem?